### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by Dukezzz12** [getlantern/lantern-forum-cn#749] [General]
昨天刚买的，网络非常不稳定，且pc端遇到bug无法打开，所以申请退款。邮箱号如下:
dukezzz1234@gmail.com
附上付款凭证

**Comment by Dukezzz12** [comment]
![Screenshot_2024-08-02-10-26-29-466_com tencent mm-edit](https://github.com/user-attachments/assets/62fdd41b-fa5f-49d7-b098-f789611ca13f)

**Comment by Dukezzz12** [comment]
请尽快处理谢谢

**Comment by MaxMKMax** [comment]
@Dukezzz12 好的. 已经帮您提交问题上去了. 他们会进行处理. 请您放心.

**Reply by Dukezzz12** [reply]
感谢，Mike
